policyengine_us/variables/gov/states/md/tax/income/credits/ctc/md_ctc.py

```diff
@@ -4,11 +4,13 @@
 class md_ctc(Variable):
     value_type = float
     entity = TaxUnit
-    label = "MD CTC"
+    label = "Maryland Child Tax Credit"
     definition_period = YEAR
     unit = USD
-    documentation = "Maryland Child Tax Credit"
-    reference = "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-751-effective-until-712026-tax-credit-for-qualified-child"
+    reference = (
+        "https://casetext.com/statute/code-of-maryland/article-tax-general/title-10-income-tax/subtitle-7-income-tax-credits/section-10-751-effective-until-712026-tax-credit-for-qualified-child"
+        "https://law.justia.com/codes/maryland/2022/tax-general/title-10/subtitle-7/section-10-751/"
+    )
     defined_for = StateCode.MD
```

Comment:
while you're here could you make `md_ctc_eligible` to improve performance? just based on agi